CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_qualtrics v0.1.1
+
+[PR #6](https://github.com/fivetran/dbt_qualtrics/pull/6) applies the following changes:
+
+## Bug Fix
+- Ensures that date spine in `qualtrics__daily_breakdown` always has a non-null start date, even if you do not have the `distribution_contact` source table, whose `min(created_at)` we base the date spine off of. 
```

Comment:
The wording of this just felt a bit off. I attempted to adjust it with the following suggestion. Let me know if you feel strongly towards keeping your original CHANGELOG entry format.

```suggestion
- Ensures the date spine in the `qualtrics__daily_breakdown` model always has a non-null start date, especially if you do not have the `distribution_contact` source table whose `min(created_at)` is used to generate the date spine. 
```